tests/e2e/cli/instance_segmentation/test_api_xai_sanity_instance_segmentation.py

```diff
@@ -184,14 +187,17 @@ def test_inference_xai(self):
             load_weights_ov = osp.join(temp_dir, "openvino.xml")
             task_env.model = read_model(task_env.get_model_configuration(), load_weights_ov, None)
             task = OpenVINODetectionTask(task_environment=task_env)
-            inference_parameters.enable_async_inference = False
-            predicted_dataset_ov = task.infer(val_dataset_copy.with_empty_annotations(), inference_parameters)
-
-            # Check saliency maps OV task
-            saliency_maps_check(
-                predicted_dataset_ov,
-                task_labels,
-                (480, 640),
-                processed_saliency_maps=processed_saliency_maps,
-                only_predicted=only_predicted,
-            )
+            for enable_async_inference in [True, False]:

```

Comment:
Could you please use the `pytest.parameterize` instead of this `for` loop? It's easier to search the log when the error occurs.

Reply:
The test case `test_inference_xai` contains several tests, so it can be split into several methods, some of which are parametrized

Reply:
Thanks for the comments. I refactored the test